src/main/java/org/killbill/billing/client/model/SubscriptionUsage.java

```diff
@@ -0,0 +1,76 @@
+package org.killbill.billing.client.model;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SubscriptionUsage  {
```

Comment:
No question - it's a good idea! But it deoends, my intention was to follow the naming of the other client model classes. We can rename the classes. 
If you want that, then it is better if I create new Pull Request?